src/test/java/com/redhat/devtools/lsp4ij/features/selectionRange/TypeScriptSelectionRangeTest.java

```diff
@@ -91,9 +94,8 @@ public void testSelectionRanges_qualifierExpression() {
 
     public void testSelectionRanges_callExpression() {
         assertSelectionRanges(
-                "demo.ts",
-                // language=typescript
-                "console.log('message');",
+                DEMO_TS_FILE_NAME,
```

Comment:
It is just my opinion, but I find using constant is not easy to read, because you need to jump to the constant to know the value.

But if you prefer , keep like this.